packages/fxa-settings/src/components/UnitRowRecoveryKey/index.test.tsx

```diff
@@ -41,4 +59,24 @@ describe('UnitRowRecoveryKey', () => {
       'Create'
     );
   });
+
+  it('can be refreshed', async () => {
+    renderWithRouter(
+      <MockedCache
+        account={{ recoveryKey: false }}
+        mocks={[mockAccountQuery({ recoveryKey: true })]}
+      >
+        <UnitRowRecoveryKey />
+      </MockedCache>
+    );
+    expect(screen.getByTestId('unit-row-header-value')).toHaveTextContent(
+      'Not Set'
+    );
+    await act(async () => {
+      fireEvent.click(screen.getByTestId('recovery-key-refresh'));
```

Comment:
I think we need account mocks in all 3 stories now to accommodate this. When you click on the refresh button on any of them it errors with "no more mocks available" and stays disabled.

Reply:
I added mocks for account recovery and 2fa, but couldn't for secondary email because it already expects mocks for verification mutations.